### ISSUE TITLE
i18n module undocumented

### ISSUE THREAD

**Issue by duncdrum** [eXist-db/documentation#266]
### Where is the problem
eXist has its own (i18n module)[eXist-db/i18n] but no documentation about its existence, or how it is supposed to work. 

@windauer as the author would you be so kind, to add documentation about this very useful module. 

### Please provide the following
*   exist-db version: `4.3.1`
*   documentation version: `4.0.3`

**Comment by adamretter** [comment]
Huh! I didn't even know this repo existed until now

**Comment by duncdrum** [comment]
@windauer a life sign please

**Comment by joewiz** [comment]
@duncdrum Are you thinking this module deserves a standalone article? (Whatever your response is, should all repos in the eXist organization register an entry in the documentation? We lack such a list, to my knowledge.)

**Comment by duncdrum** [comment]
@joewiz we are historically inconsistent some repos do have full articles (dashboard), others are listed with a short description and a link, others not mentioned at all, others have their own documentation (exide). 

since we have a diverse group of repos in the org, org membership is not a good criteria. It's more what they do and how they fit into the current documentation structure. And this seems highly relevant to `application development` which is no freely available elsewhere on the web, and our biggest section of articles. Thats why i think it should appear in here.

This can be a link to a readme or gh-page with fuller documentation, an article, or… but something. Happy to help find a good spot for it.

**Comment by windauer** [comment]
sorry for staying quiet so long.. I was kind of offended from the initial ticket to be honest. Why? Because:  http://letmegooglethat.com/?q=exist-db+i18n

1. [eXist i18n XQuery Module Documentation](https://exist-db.org/exist/apps/demo/examples/special/i18n-docs.html) and [source](https://github.com/eXist-db/demo-apps/blob/master/examples/special/i18n-docs.html)
2. [FunDocs](https://exist-db.org/exist/apps/fundocs/view.html?uri=http://exist-db.org/xquery/i18n&location=/db/apps/demo/modules/i18n.xql)
3. [Original Source](https://github.com/eXist-db/i18n)

From my perspective the module is a bit outdated. Would love to spend some time on it and even had some productive discussions with @wolfgangmm on how to improve it  but I currently I'm lacking the time and a budget.

**Comment by duncdrum** [comment]
Yes so google knows, supposes that a) users have to know what to search for its not a replacement for having links or mentions in our official docs. 
The funcdocs page is good, but doesn't explain how users can use this, it also needs to be installed locally to display that info. 
As we discusses multiple times during the time of the OP and the large rewrite of the docs, having some stuff in demo, and other in the docs just means that folks miss out. Even if outdated the version from demo is still current for the state of the app. So can you prepare a PR that a) either migrates the contents, or adds section with a link and some one-liner about the module to the docs.
